keras_cv/layers/preprocessing/base_image_augmentation_layer.py

```diff
@@ -235,20 +235,20 @@ def call(self, inputs, training=True):
     def _augment(self, inputs):
         image = inputs.get(IMAGES, None)
         label = inputs.get(LABELS, None)
-        bounding_box = inputs.get(BOUNDING_BOXES, None)
+        bounding_boxes = inputs.get(BOUNDING_BOXES, None)
         transformation = self.get_random_transformation(
```

Comment:
hey Divya, can you update _augment to introduce the following:
```
    def _augment(self, inputs):
        image = inputs.get(IMAGES, None)
        label = inputs.get(LABELS, None)
        bounding_box = inputs.get(BOUNDING_BOXES, None)
        transformation = self.get_random_transformation(
            image=image, label=label, bounding_boxes=bounding_box
        )
        image = self.augment_image(image, transformation=transformation, bounding_boxes=bounding_box, label=label)
        result = {IMAGES: image}
        if label is not None:
            label = self.augment_target(label, transformation=transformation, bounding_boxes=bounding_box, label=label, image=image)
            result[LABELS] = label
        if bounding_box is not None:
            bounding_box = self.augment_bounding_boxes(
                bounding_box, transformation=transformation, bounding_boxes=bounding_box, label=label, image=image
            )
            result[BOUNDING_BOXES] = bounding_box
        return result
```

This includes the other label types into the other augment_* methods.